src/test/java/api/loans/scenarios/CheckInDeclaredLostItemTest.java

```diff
@@ -0,0 +1,484 @@
+package api.loans.scenarios;
+
+import static api.support.http.CqlQuery.queryFromTemplate;
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.everyItem;
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.CoreMatchers.startsWith;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.folio.circulation.domain.FeeAmount;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.hamcrest.Matcher;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Before;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.MultipleJsonRecords;
+import api.support.builders.CheckInByBarcodeRequestBuilder;
+import api.support.builders.DeclareItemLostRequestBuilder;
+import io.vertx.core.json.JsonObject;
+
+public class CheckInDeclaredLostItemTest extends APITests {
+  private static final String LOST_ITEM_FEE = "Lost item fee";
+  private static final String LOST_ITEM_PROCESSING_FEE = "Lost item processing fee";
+  private static final String CANCELLED_ITEM_RETURNED = "Cancelled item returned";
+  private static final String REFUNDED_FULLY = "Refunded fully";
+  private static final String CREDITED_FULLY = "Credited fully";
+  private static final String LOST_ITEM_FOUND = "Lost item found";
+
+  private IndividualResource item;
+  private IndividualResource loan;
+
+  @Before
+  public void activateChargeableLostItemFeePolicy() {
+    useLostItemPolicy(lostItemFeePoliciesFixture.chargeFee().getId());
+  }
+
+  @Test
+  public void canCancelItemFeeOnly() {
+    final double itemFee = 15.00;
+
+    useLostItemPolicy(lostItemFeePoliciesFixture.create(
+      lostItemFeePoliciesFixture.facultyStandardPolicy()
+        .withName("Test check in")
+        .doNotChargeProcessingFee()
```

Comment:
Is this test demonstrating that the item fee should be cancelled when there is only an item fee and no processing fee?

Reply:
yes, correct.

Reply:
Then I think maybe a better name might be `shouldCancelItemFeeOnlyWhenNoOtherFeesCharged`. What do you think?

(and similarly for when there is only a processing fee)

Reply:
renamed for both cases.